Change stats symbol

jadebuckwalter opened the pull request:
Replace the "i" icon with a bar graph symbol.

jadebuckwalter committed:
Change stats symbol

Replace the "i" icon with a bar graph symbol.

jadebuckwalter commented:
I also changed the color of the toolkit from green to black as we discussed in last week's meeting.

psvenk commented:
Thank you @jadebuckwalter for the change. I think it would be better to use the "i" icon next to each assignment while using the bar graph icon in the header, as this would coordinate better with the symbolism of the toolbox and hammer (the toolbox better indicates the action while the hammer communicates something auxiliary or subordinate), removes the need for the inverted icon (with the regular icon looking more harmonious with the toolbox icon), and also is consistent with #167 once implemented. Your thoughts?

jadebuckwalter commented:
@psvenk Yes, definitely! The inverted bar graph looks a little strange anyway once it’s shrunken down to that size.

psvenk committed:
Use "i" icon for assignments; remove inverted icon

Using the "i" icon for the stats button on individual assignments
coordinates better with the symbolism of the toolbox and hammer for test
corrections (the toolbox better indicates the action while the hammer
communicates something auxiliary or subordinate), is consistent with
issue #167 (show information for assignments without statistics), and
also removes the inverted bar graph icon which looked a bit out of
place.
